pkg/apis/serving/v1alpha2/framework_tensorflow_test.go

```diff
@@ -129,3 +129,52 @@ func TestTensorflowContainer(t *testing.T) {
 	container := spec.GetContainer("someName", 0, &config)
 	g.Expect(container).To(gomega.Equal(expectedContainer))
 }
+
+func TestTensorflowGRPC(t *testing.T) {
```

Comment:
nice test! let's add an e2e test as well?

Reply:
It looks like the e2e test is using the sdk. Do you know how can I generate the sdk code using my new protocol spec?